scriptworker/cot/verify.py

```diff
@@ -29,15 +29,19 @@
 from scriptworker.constants import DEFAULT_CONFIG
 from scriptworker.context import Context
 from scriptworker.exceptions import CoTError, BaseDownloadError, ScriptWorkerGPGException
+from scriptworker.github import GitHubRepository
 from scriptworker.gpg import get_body, GPG
 from scriptworker.log import contextual_log_handler
 from scriptworker.task import (
+    extract_github_repo_owner_and_name,
     get_action_callback_name,
     get_and_check_project,
     get_and_check_tasks_for,
     get_commit_message,
     get_decision_task_id,
     get_parent_task_id,
+    get_pull_request_number,
+    get_push_date_time,
```

Comment:
These functions seem like they equally belong in `scriptworker.github` and `scriptworker.task`. If we end up supporting, say, bitbucket as well, and if it has different behavior, we might end up moving these functions.